.travis.yml

```diff
@@ -1,27 +1,34 @@
 language: generic
-sudo: required
+sudo: false
 dist: trusty
```

Comment:
Consider using an updated distro since the sudo isn't holding back anymore.

Reply:
the newest one available to travis is trusty. (unless that's not what you meant?)

Reply:
Well then.

Reply:
Sadly I can't figure out what "rebase the last commit on top of the updated master branch" means.    
Can you translate that into git commands for me?
By a clean pull request, do you mean a new one?

Reply:
Roughly the course of action should be the following:

```
git remote add upstream https://github.com/sakhnik/gpwsafe
git fetch upstream
git rebase upstream/master     # Resolve conflicts if any
```

Check the following now:

```
git log upstream/master..
```

That many commits will show up in the PR after you do `git push`.